File: Server/src/main/content/global/skill/production/cooking/recipe/pizza/impl/AnchovyPizza.java
package content.global.skill.production.cooking.recipe.pizza.impl;

import content.global.skill.production.cooking.recipe.pizza.PizzaRecipe;
import core.game.node.item.Item;

/**
 * The Anchovy pizza.
 */
public class AnchovyPizza extends PizzaRecipe {
    private static final Item ANCHOVY_PIZZA = new Item(2297);
    private static final Item ANCHOVIES = new Item(319);

    @Override
    public double getExperience() {
        return 39;
    }

    @Override
    public Item getProduct() {
        return ANCHOVY_PIZZA;
    }

    @Override
    public Item[] getIngredients() {
        return new Item[]{ANCHOVIES};
    }

    @Override
    public int getLevel() {
        return 55;
    }

}

File: Emulator/src/main/content/global/skill/cooking/type/pizza/impl/AnchovyPizza.kt
package content.global.skill.cooking.type.pizza.impl

import content.global.skill.cooking.type.pizza.PizzaRecipe
import org.rs.consts.Items
import core.game.node.item.Item

/**
 * Anchovy pizza.
 */
class AnchovyPizza : PizzaRecipe() {

    override fun getExperience(): Double {
        return 39.0
    }

    override fun getProduct(): Item {
        return ANCHOVY_PIZZA
    }

    override fun getIngredients(): Array<Item> {
        return arrayOf(ANCHOVIES)
    }

    override fun getLevel(): Int {
        return 55
    }

    companion object {
        private val ANCHOVY_PIZZA = Item(Items.ANCHOVY_PIZZA_2297)
        private val ANCHOVIES = Item(Items.ANCHOVIES_319)
    }
}
